client/src/components/scheduler/Overview.js

```diff
@@ -23,6 +23,18 @@ const Overview = ({ name, duration, description }) => {
         <AccessTime />
         <Typography variant="caption">{duration} minute</Typography>
       </Grid>
+      {appointmentTime ? (
+        <Grid
+          container
+          item
+          alignItems="center"
+          className={classes.iconWrapper}
+        >
+          <Typography variant="caption">
+            {JSON.stringify(appointmentTime)}
```

Comment:
Interesting use of `JSON.stringify`. Since we're dealing with a boolean, not a JSON object, it may technically be more correct to use `appointmentTime.toString()`

Reply:
Ok! This was mostly some placeholder until I formatted the times in my next PR but good to know